readthedocs/doc_builder/python_environments.py

```diff
@@ -43,15 +43,20 @@ def delete_existing_build_dir(self):
     def install_package(self):
         setup_path = os.path.join(self.checkout_path, 'setup.py')
         if os.path.isfile(setup_path) and self.config.install_project:
-            if self.config.pip_install or getattr(settings, 'USE_PIP_INSTALL', False):
+            if self.config.pip_install or getattr(
+                    settings, 'USE_PIP_INSTALL', False):
+                extra_req_param = ''
+                if self.config.extra_requirements:
+                    extra_req_param = '[{0}]'.format(
+                        ','.join(self.config.extra_requirements))
                 self.build_env.run(
                     'python',
                     self.venv_bin(filename='pip'),
                     'install',
                     '--ignore-installed',
                     '--cache-dir',
                     self.project.pip_cache_path,
-                    '.',
+                    '.{0}'.format(extra_req_param),
```

Comment:
As decided in https://github.com/rtfd/readthedocs.org/pull/2368#discussion_r75264711 , we'll keep the `{0}` format.